variables.tf

```diff
@@ -1,11 +1,127 @@
-##############################################################################
-# Input Variables
-##############################################################################
+variable "resource_group_id" {
+  description = "ID of resource group to use when creating the event stream instance"
+  type        = string
+}
 
-#variable "myvariable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = ""
-#}
+variable "es_name" {
+  type        = string
+  description = "The name to give the IBM Event Streams instance created by this module."
+}
 
-##############################################################################
+variable "plan" {
+  type        = string
+  description = "Plan for the event streams instance : lite, standard or enterprise-3nodes-2tb"
+  default     = "standard"
+  validation {
+    condition     = contains(["lite", "standard", "enterprise-3nodes-2tb"], var.plan)
+    error_message = "The specified plan is not a valid selection! Supported plans are: lite, standard or enterprise-3nodes-2tb."
+  }
+}
+
+variable "tags" {
+  type        = list(string)
+  description = "List of tags associated with the Event Steams instance"
+  default     = []
+}
+
+variable "region" {
+  type        = string
+  description = "IBM Cloud region where event streams will be created"
+  default     = "us-south"
+}
+
+variable "throughput" {
+  type        = number
+  description = "Throughput capacity in MB per second. for enterprise instance only. Options are: 150, 300, 450. Default is 150."
+  default     = "150"
+}
+
+variable "storage_size" {
+  type        = number
+  description = "Storage size of the event streams in GB. for enterprise instance only. Options are: 2048, 4096, 6144, 8192, 10240, 12288. Default is 2048.      Note: When throughput is 300, storage_size starts from 4096,  when throughput is 450, storage_size starts from 6144."
```

Comment:
there seems to be multiple spaces used after some of the full stops here. can you clean up?